### ISSUE TITLE
Revert the detailed trajectories

### ISSUE THREAD

**Issue by asiripanich** [asiripanich/emdash#40]
Quick fix for #29.

**Comment by shankari** [comment]
FYI, while testing the updates, the `emdash` container appeared to run out of memory and get killed on my laptop right after loading locations. This was with 3 months worth of data.

```
Listening on http://0.0.0.0:8010
Warning in register_theme_dependency(func) :
  This functionality requires shiny v1.6 or higher
Warning in get_current_theme() :
  This functionality requires shiny v1.6 or higher
Warning in register_theme_dependency(func) :
  This functionality requires shiny v1.6 or higher
Warning in get_current_theme() :
  This functionality requires shiny v1.6 or higher
Warning in register_theme_dependency(func) :
  This functionality requires shiny v1.6 or higher
Warning in get_current_theme() :
  This functionality requires shiny v1.6 or higher
Warning in register_theme_dependency(func) :
  This functionality requires shiny v1.6 or higher
Warning in get_current_theme() :
  This functionality requires shiny v1.6 or higher
Running: mod_load_data_server
About to load trips
Finished query, about to clean trips
Finished cleaning trips
Finished loading trips
About to load server calls
Finished query, about to tidy server calls
Finished tidying server calls
Finished loading server calls
About to load locations
Finished query, about to clean locations
Killed
```

**Comment by shankari** [comment]
Interestingly, after the fixes in #43, the dashboard loads, and the participant table works correctly, but the map view still kills the R process. Even without the trajectories. I'm going to disable the map view in my current deployments.

```
Finished loading participants
About to load participants
merging
Finished loading participants
Warning in grSoftVersion() :
  unable to load shared object '/usr/local/lib/R/modules//R_X11.so':
  libXt.so.6: cannot open shared object file: No such file or directory
Warning: Removed 15 rows containing non-finite values (stat_count).
Warning in register_theme_dependency(func) :
  This functionality requires shiny v1.6 or higher
... (repeated warning many many times)
Warning in register_theme_dependency(func) :
  This functionality requires shiny v1.6 or higher
Warning in get_current_theme() :
  This functionality requires shiny v1.6 or higher
Killed
```
